Source/ChilliSource/Core/State/State.h

```diff
@@ -308,19 +317,6 @@ namespace ChilliSource
         //-----------------------------------------
         virtual void OnFixedUpdate(f32 in_deltaTime){};
         //-----------------------------------------
-        /// The render snapshot event can be
-        /// implemented by a state to allow it to
-        /// snapshot any data which pertains to the
-        /// renderer.
-        ///
-        /// @author Ian Copland
-        ///
-        /// @param in_renderSnapshot - The render
-        /// snapshot object which contains all
-        /// snapshotted data.
-        //-----------------------------------------
-        virtual void OnRenderSnapshot(class RenderSnapshot& in_renderSnapshot) noexcept {};
```

Comment:
I removed it because of the ambiguity surrounding multiple render targets (the confusion still kind of exists for systems). It's not obvious whether this is called for all render snapshots or just the main one. Should we call it for all and have some way of distinguising the render targets?